### PR TITLE
chore: 🤖 update changelog and version for v2.0.0

### PR DESCRIPTION
**Pull request by lisbet-alvarez**
## Description
Add changelog updates and update cli version for 2.0.0 DC release

### CODE REVIEW

**Review by cameronperera — APPROVED** [review]
LGTM

**Review by DhariniJeeva — APPROVED** [review]

File: ui/desktop/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
+
+### New
+
+- Upgrade CLI to v0.15.0 ([PR](https://github.com/hashicorp/boundary-ui/pull/2117))
+- Add search, filtering, and pagination support for sessions and targets ([PR](https://github.com/hashicorp/boundary-ui/pull/2091))
+- Add time-remaining to a session ([PR](https://github.com/hashicorp/boundary-ui/pull/2018))
+
+### Bug Fixes
+- Fix copy command not working in embedded terminal for Windows OS ([PR](https://github.com/hashicorp/boundary-ui/pull/2001))
+- Fix broken hot reloading ([PR](https://github.com/hashicorp/boundary-ui/pull/1992))
```
Comment (ZedLi) [inline ui/desktop/CHANGELOG.md:15]: Just saw this, do we typically include dev only changes/bug fixes?

Comment (cameronperera) [reply ui/desktop/CHANGELOG.md:15]: No we don't. I should have looked at that PR. I thought it was something else for some reason.

Comment (lisbet-alvarez) [reply ui/desktop/CHANGELOG.md:15]: are we only referring to the bug fix on line 15 ?

Comment (ZedLi) [reply ui/desktop/CHANGELOG.md:15]: Yup

**Pull request merged**